File: UKOM/Moview_backend/app/Models/MovieRelease.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Model;

class MovieRelease extends Model
{
    protected $fillable = [
        'movie_id',
        'type',
        'country_code',
        'name',
        'release_date',
    ];

    protected $casts = [
        'release_date' => 'date',
    ];

    public const TYPE_PREMIERE = 'premiere';
    public const TYPE_THEATRICAL = 'theatrical';
    public const TYPE_STREAMING = 'streaming';

    public function movie()
    {
        return $this->belongsTo(Movie::class);
    }
}

File: UKOM/Moview/app/src/main/java/com/komputerkit/moview/data/model/MovieRelease.kt
package com.komputerkit.moview.data.model

data class MovieRelease(
    val type: String,
    val typeLabel: String,
    val countryCode: String? = null,
    val countryName: String? = null,
    val flagEmoji: String? = null,
    val name: String? = null,
    val releaseDate: String? = null
)
